README.md

```diff
@@ -1,17 +1,17 @@
 Cat2Cat (Catalog to Category)
 =========
 ----
-`Cat2Cat` exists to help solve the typo problem with UIImage's `imageNamed:` method. 
```

Comment:
Is there any way to update the diff on this file so that it's only changing the bits which have actually changed and not just tweaking whitespace?